pkg/api/dtos/frontend_settings.go

```diff
@@ -56,11 +56,17 @@ type FrontendSettingsLicenseInfoDTO struct {
 	AppUrl      *string `json:"appUrl,omitempty"`
 }
 
+type FrontendSettingsAzureCloudDTO struct {
+	Name        string `json:"name"`
+	DisplayName string `json:"displayName"`
+}
+
 type FrontendSettingsAzureDTO struct {
-	Cloud                   string `json:"cloud"`
-	ManagedIdentityEnabled  bool   `json:"managedIdentityEnabled"`
-	WorkloadIdentityEnabled bool   `json:"workloadIdentityEnabled"`
-	UserIdentityEnabled     bool   `json:"userIdentityEnabled"`
+	Cloud                   string                          `json:"cloud"`
+	Clouds                  []FrontendSettingsAzureCloudDTO `json:"clouds"`
```

Comment:
This type should also be updated in `@grafana/runtime` so that there's no need to extend the type in the SDK PR

Reply:
Correct, expansion in the frontend SDK PR is a temporary workaround until `@grafana/runtime` updated.

Reply:
PR for `@grafana/runtime`: https://github.com/grafana/grafana/pull/84039